java/runtime/src/main/java/org/ray/runtime/WorkerContext.java

```diff
@@ -63,18 +63,15 @@ public UniqueId getCurrentTaskId() {
    * Set the current task which is being executed by the current worker. Note, this method can only
    * be called from the main thread.
    */
-  public void setCurrentTask(TaskSpec task, ClassLoader classLoader) {
+  public void setCurrentTask(UniqueId currentTaskId,
```

Comment:
Set `currentTaskId` to `UniqueId.NIL` if it's `null`?

Reply:
right, but maybe it's more clear to just pass in `UniqueId.NIL`